session_csrf/__init__.py

```diff
@@ -3,9 +3,14 @@
 import hashlib
 
 from django.conf import settings
+from django import VERSION as DJANGO_VERSION
 from django.core.cache import cache
 from django.middleware import csrf as django_csrf
-from django.utils import crypto
+try:
+    from django.middleware.csrf import _get_new_csrf_key as django_get_new_csrf_string
+except ImportError:
+    from django.middleware.csrf import _get_new_csrf_string as django_get_new_csrf_string
```

Comment:
Is there really no public function available in `django.middleware.csrf` to get a token string out?!